### DISCUSSION TITLE
Pinecil V2 changes on schematic

### DISCUSSION THREAD

**Discussion by River-Mochi** [Ralim/IronOS#1390] [Q&A] [answered]
reddit post  says there is, "good reason [Pine Store] not advertising 28V support" because filtering caps on the V2 schematic look like they were downspeced from 50V to 25V.  

Is there any validity to this in relation to safely using EPR 28V chargers?

post says: _"BTW. According to the [v2 production schematic](https://files.pine64.org/doc/Pinecil/Pinecil_schematic_v2.0_20220608.pdf) all input filtering caps have been downspeced from 50V to 25V. So there is a [good reason they are not advertising 28V support. ](https://www.reddit.com/r/PINE64official/comments/wxjaug/comment/ilubidw/?context=3)"_

 I think Thanos said in that capacitors will be fine past 25V as long as you don't hit them with Reverse polarity ( and that only happens if someone plugs in random old DC barrel charger without first checking if center pin is positive. that would blow up either V1 or V2. we saw this happen on V1 even with pcb parts it had)

**Comment by Ralim** [comment, marked as answer]
Parts are rated to 35V according to manufacturer. Unsure why schematic denotes as 25V. Still trying to find out whole story.

**Reply by River-Mochi** [reply]
thank you for checking on this.
Also the schematic V2  shows U14  as  populated (hall sensor).   On V1 older schematic  it says (Not populated).  

-- would be good if Pine64 could fix that because we already had people asking if V2 is coming with hall sensor since schematic missing words (not populated).   and it's not there.

**Reply by Ralim** [reply]
So they are going to update schematic. 

Samsung sales Rep confirmed. Caps are rated to 42V max, so 28 is fine.


Have also asked to update Hall effect.
